recipes/faampy/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "faampy" %}
+{% set version = "v0.1.4" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: faampy-{{ version }}.tar.gz
+    url: https://github.com/ncasuk/faampy/archive/{{ version }}.tar.gz
+    sha256: 9540646948656051d484e17b0df2fc02c83eff411137f956a7a2f523e0eb30dc
+
+build:
+    skip: True  # [py3k]
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+
+    run:
+        - python
+        - basemap
+        - cartopy
+        - configparser
+        - ephem
+        - matplotlib >=1.4.0
+        - setuptools
+        - netcdf4 >=1.0
+        - numpy
+        - scipy
+        - pandas
+        - gdal
+        - exifread
+        - fiona
+        - gdal
+        - georaster
+        - haversine
+        - pillow
+        - libspatialite
+        - seaborn
+        - simplekml
+        - sqlite
```

Comment:
Please pin `sqlite` to `3.13.*`. See all the pinnings in https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L31-L73